climakitae/data/variable_descriptions.csv

```diff
@@ -1,4 +1,5 @@
 variable_id,downscaling_method,timescale,unit,display_name,extended_description,colormap,show
+fosberg_index_derived,Dynamical,hourly,[0 to 100],Fosberg fire weather index,"Fire weather index using meteorological inputs (temperature, relative humidity, and windspeed) to quantify fire risk.",ae_orange,TRUE

```

Comment:
Wondering if we should match the extended description further to match the FFWI reference more?

Something like "Fire weather index using meteorological inputs (temperature, relative humidity, windspeed) to quantify fire risk. Large values imply a higher risk value."

Reply:
Love it. Just updated